src/app/services/usuario.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { environment } from 'src/environments/environment';
+import { tap} from 'rxjs/operators';
+import { Router } from '@angular/router';
+const base_url = environment.apiUrl;
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UsuarioService {
+
+  constructor(private http: HttpClient,
+    private router: Router) { }
+
+  login( formData: any ) {
+    
+    return this.http.post(`${ base_url }/login`, formData ).pipe(
+      tap( (resp: any) => {
+        localStorage.setItem('token', resp.token )
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7127387)
```suggestion
        localStorage.setItem("token", resp.token )
```